chrome/content/zotero/xpcom/style.js

```diff
@@ -347,8 +347,9 @@ Zotero.Styles = new function() {
 			
 			if(existingFile) {
 				// find associated style
-				for (let existingStyle of _styles) {
-					if(destFile.equals(existingStyle.file)) {
+				for (let id in _styles) {
+					let existingStyle = _styles[id];
+					if(existingStyle.file && destFile.equals(existingStyle.file)) {
```

Comment:
With the other changes in place, do you have a case when this fails without the `existingStyle.file` check?